src/aequitas/flow/experiment/experiment.py

```diff
@@ -258,3 +259,14 @@ def dt_handler(x):
             json.dumps(self.config, default=dt_handler, sort_keys=True).encode("utf-8")
         ).hexdigest()
         self.logger.debug(f"Hash generated: {self.hash}.")
+
+    def plot_pareto(
+        self,
+        dataset: str,
+        fairness_metric: str = "Predictive Equality",
+        performance_metric: str = "TPR",
+        split: str = "validation",
+    ) -> None:
+        results = read_results(self.exp_folder)
+        plot = Plot(results, dataset, fairness_metric, performance_metric, split=split)
+        return plot.visualize()
```

Comment:
I would rather store the pareto plot, in case people want to use the disparities or summary plot too :pray: